resqpy/well/_wellbore_marker_frame.py

```diff
@@ -349,32 +294,10 @@ def create_xml(self,
             log.error('trajectory object is missing and must be included')
 
         # fill wellbore marker
-        for marker in self.wellbore_marker_list:
-
-            wbm_node_obj = self.model.new_obj_node('WellboreMarker', is_top_lvl_obj = False)
-            wbm_node_obj.set('uuid', marker[0])
-            wbm_node.append(wbm_node_obj)
-            wbm_gb_node = rqet.SubElement(wbm_node_obj, ns['resqml2'] + 'GeologicBoundaryKind')
-            wbm_gb_node.set(ns['xsi'] + 'type', ns['xsd'] + 'string')
-            wbm_gb_node.text = str(marker[1])
-
-            interp = marker[3]
-            if interp is not None:
-                interp_root = marker[3].root
-                if 'HorizonInterpretation' in str(type(marker[3])):
-                    self.model.create_ref_node('Interpretation',
-                                               rqet.find_tag(rqet.find_tag(interp_root, 'Citation'), 'Title').text,
-                                               bu.uuid_from_string(interp_root.attrib['uuid']),
-                                               content_type = 'obj_HorizonInterpretation',
-                                               root = wbm_node_obj)
-
-                elif 'FaultInterpretation' in str(type(marker[3])):
-                    self.model.create_ref_node('Interpretation',
-                                               rqet.find_tag(rqet.find_tag(interp_root, 'Citation'), 'Title').text,
-                                               bu.uuid_from_string(interp_root.attrib['uuid']),
-                                               content_type = 'obj_FaultInterpretation',
-                                               root = wbm_node_obj)
-
+        if self.marker_list is not None:  # TODO: confirm that you can have an empty wellbore marker frame with no marker list
+            for marker in self.marker_list:
+                wbm_node_obj = marker.create_xml(parent_node = wbm_node, title = 'wellbore marker')
```

Comment:
We could consider putting a more complicated title in here, something including a well name and feature name.

Reply:
something like this potentially? 
**title = str(self.trajectory.well_name) + '_' + marker.marker_type**